docs/updating/4_update_2.2.md

```diff
@@ -103,24 +107,41 @@ you have an option to continue, but migrated Landing Pages will come without tho
+Landingpage blocks (2.1 and earlier) was defined using a class implementing `EzSystems\LandingPageFieldTypeBundle\FieldType\LandingPage\Model\AbstractBlockType`. 
+In pagebuilder (2.2), this interface is no longer present. Instead the logic of your block must be implemented in a [Listener](../guide/extending/extending_page.md#block-rendering-events).
+Typically, what you previously would do in `getTemplateParameters()`, you'll now do in the `onBlockPreRender()` event handler.
 
-You also need to redefine [YAML configuration](../guide/extending/extending_page.md#creating-page-blocks) for your custom blocks.
+The definition of block parameters has to be moved from `createBlockDefinition()` to the [YAML configuration](../guide/extending/extending_page.md#creating-page-blocks) for your custom blocks.
 
-!!! caution
+More information about how custom blocks are implemented in Pagebuilder, please have a look at the [documentation](../guide/extending/extending_page.md)
 
-    Since v2.2 you no longer need to use services for custom Page Blocks, you can create them using YAML configuration.
+For the migration of blocks from landingpage you Pagebuilder, you'll need to provide a converter for attributes of custom blocks. For simple blocks you can use `\EzSystems\EzPlatformPageMigration\Converter\AttributeConverter\DefaultConverter`.
+Custom converters must implement the `\EzSystems\EzPlatformPageMigration\Converter\AttributeConverter\ConverterInterface` interface.
+`convert()` will parse XML `\DOMNode $node` and return an array of `\EzSystems\EzPlatformPageFieldType\FieldType\LandingPage\Model\Attribute` objects.
 
-The service definition has to be tagged as:
+Below is an example of a simple converter for a custom block:
 
 ``` yaml
-tags:
-    - { name: ezplatform.fieldtype.ezlandingpage.migration.attribute.converter, block_type: my_block_type_identifier }
+    app.block.foobar.converter:
+        class: EzSystems\EzPlatformPageMigration\Converter\AttributeConverter\DefaultConverter
+        tags:
+            - { name: ezplatform.fieldtype.ezlandingpage.migration.attribute.converter, block_type: foobar }
 ```
 
-Custom converters must implement the `\EzSystems\EzPlatformPageMigration\Converter\AttributeConverter\ConverterInterface` interface.
-`convert()` will parse XML `\DOMNode $node` and return an array of `\EzSystems\EzPlatformPageFieldType\FieldType\LandingPage\Model\Attribute` objects.
+Notice the service tag that must be used for attribute converters.
+
+This converter is only needed when running the `ezplatform:page:migrate` script and can be removed once that has completed.
+
+#### Example on how to migrate layouts and blocks to the new Page Builder
+
+In the `upgraded_to_2.2` branch in this [git repo](https://github.com/ezsystems/ezplatform-ee-beginner-tutorial/tree/upgraded_to_2.2) you may see an example on how to migrate your source code in order to make a layout and a block work with the new Page Builder.
```

Comment:
How about adding the `src/AppBundle/Resources/config/blocks.yml` and `/Event/Listener/RandomBlockListener.php` from the example here instead of linking? To make the page less cluttered, we could place it in an expanding code box.

Reply:
I added the code in expanding code box. Please have a look